core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AutoscaleV4Controller.java

```diff
@@ -94,6 +98,28 @@ public void putStack(@TenantAwareParam @ResourceCrn String crn, String userId, @
         stackCommonService.putInDefaultWorkspace(crn, updateRequest);
     }
 
+    @Override
+    @CheckPermissionByResourceCrn(action = AuthorizationResourceAction.SCALE_DATAHUB)
```

Comment:
Do we want to use the same scale right for this start API?

Reply:
If they don't wish to have 2 separate personas configuring this independently, I'm ok with it. Althogh UMS teams suggestion is to be as specific as possible.

Reply:
Yes. This is considered an alternate mechanism for scaling.